### PR TITLE
Rename some arm guards to vambrace and create new vambraces/arm guards fixes #57584

### LINKED ISSUE #57584: Inconsistent coverage between arm guards and vambraces
### Describe the bug

There are multiple items that are named "arm guard". However some of them cover the complete arm (upper arm, lower arm and elbows), while some only cover the lower arms or only the lower arms and the elbows. Similarly there are vambraces that cover only the lower arms and some cover the elbows as well. I will list some examples:

arm guards (examples)
2-by-arm guards: lower arms
chitin arm guards: lower arms and elbows
hard arm guards: upper arms, lower arms and elbows
leather arm guards: lower arms and elbows

vambraces (I think these are all currently existing):
bronze vambraces (lower arms)
leather vambraces (lower arms)
Hub 01: (lower arms and elbows)

### Steps to reproduce

level tailoring/fabrication
search for arm guards and vambraces and check coverage

### Expected behavior

I think ideally everything that only covers the lower arms (and maybe elbows) should be named vambrace everything that covers the whole arm should be an arm guard, so it can be expected what the armor covers from the name and also armor specifically covering only the lower arms are easier searchable in the crafting menu.
So the "2-by-arm guards" should be "2-by-vambraces" same with the chitin arm guards. For the leather arm guards there are already arm guards and vambraces, but the only difference is elbow coverage. So maybe the leather arm guards should just also cover the upper arm instead? I also don't really understand the stats for leather arm guards, as they seem to have better protection values for less encumbrance than vambraces.

### Screenshots

_No response_

### Versions and configuration

experimental from yesterday

### Additional context

_No response_

### PR DESCRIPTION
**Pull request by Balthasar-eu**
#### Summary
Content "Add two new vambraces and arm guards and rename existing arm guards into vambraces if coverage is only lower arms and/or elbows"

#### Purpose of change

This aims to bring more consistency to the naming of arm protection as outlined in #57584
I did not want to remove any existing armor, but I would like to have arm armor easily searchable by coverage, as there is torso armor with and without upper arm coverage it is important to be able to actually find arm armor that fits to existing torso armor.

#### Describe the solution

This pull requests adds a few chitin, leather and 2x4 arm guards and renames the existing armor to vambraces. It also renames leather vambraces to braces and reduces encumbrance a bit, as it was higher than for the arm guards with less coverage from the same material.

Added:

- Leather (full coverage) arm guard 
- Chitin (full coverage) arm guard
- Hard vambraces
- Metal vambraces

Renamed:

- Leather arm guard → Leather vambraces
- Leather vambraces → Leather bracers
- Chitin arm guards → Chitin vambraces
- 2byarm guards → 2byvambraces
- scrap arm guards → scrap vambraces
- paper arm guards → paper vambraces
- improvised tire arm guards → improvised tire vambraces

Changed (other than name):

- reduced leather bracer encumbrance
- reduced bronze vambraces volume from over 5L to 1.5L to match existing vambraces, weight was kept the same
- updated itemgroups for chitin and leather armguards to use vambraces

#### Describe alternatives you've considered

Leave it as it is.

#### Testing

I opened a game added skill and looked at the crafting menu. No game errors and the values are as expected.

### CODE REVIEW

**Comment by Balthasar-eu** [comment]
Yeah, I agree the 2x4 arm guards with upper arm protection might not be that useful and I can remove it from the PR.

Also currently there are only three new items with variants, with 2 more that I want to add, which I don't think is excessive. i don't think it makes sense to add vambraces for non rigid materials like chainmail or neoprene, but for hard materials vambraces make sense and for leather and chitin there is no standalone full arm version, which also make sense to exist.

2x4 arm guard (full arm)
leather arm guard (full arm)
chitin arm guard (full arm)

to be added:
plastic vambrace
steel vambrace

**Comment by Balthasar-eu** [comment]
Okay, I think it should be good to go now.

File: data/json/items/armor/arms_armor.json [hunk 1]
```diff
@@ -934,22 +1069,123 @@
...
+    "flags": [ "OUTER", "BLOCK_WHILE_WORN", "WATER_FRIENDLY" ],
+    "armor": [
+      {
+        "covers": [ "arm_l", "arm_r" ],
+        "encumbrance": 6,
+        "coverage": 95,
+        "cover_melee": 95,
+        "cover_ranged": 90,
+        "cover_vitals": 70,
+        "specifically_covers": [ "arm_lower_r", "arm_lower_l", "arm_upper_r", "arm_upper_l" ]
```
Comment (bombasticSlacks) [inline data/json/items/armor/arms_armor.json:1168]: This contradicts the language you are trying to standardize

**Comment by bombasticSlacks** [comment]
So your PR description doesn't match what you've actually added so I might be missing something.

Speaking in general a bunch of these don't make sense. Lots of things like bug chitin only go to the elbow because the material isn't bonded to anything it's a single piece, you couldn't move your joints if it went onto the upper arm.

Also most the current lower + upper arm armor is that way because its just two bits of basically garbage tied to your arms with string. Upper + Lower should be poor encumbrance bad options almost exclusively cause having shit that isn't hooked to anything tied to your upper arms is an awful idea.

In general upper arm armor should come from a torso piece or excessive fabrication (and even then that fabrication should lead to integrating it with torso armor) since otherwise how is it staying attached. Modern manufactured stuff is different like the current hard arm guards, but making something that functions over a joint (elbow) on both sides is hard.

File: data/json/items/armor/arms_armor.json (hunk 1 above)
Comment (Balthasar-eu) [reply data/json/items/armor/arms_armor.json:1168]: Yeah, I forgot to change the coverage after copying from the metal arm guard

**Comment by Balthasar-eu** [comment]
I increased chitin and leather arm guard encumbrance by 2 each and reduced the protection values for the elbow in the leather armguard to simulate more flexible material in the joint. For the chitin armguard though, joint articulation was already a required proficiency, so I don't think even the old armguard was supposed to be a single solid piece. And I agree it is hard, but that is what the proficiency is for, from my understanding.
I also agree to the other points in principle, but there is already precedence for armor staying attached to the upper arm with scrap or tire armguards and all the steel arm guards. I don't think it should be a problem to secure leather armguards with straps to the arm if it is not a problem for tire or metal armguards. I also don't mind increasing encumbrance more, but that would not make sense when existing full coverage steel arm guards have only 10 encumbrance.

**Comment by bombasticSlacks** [comment]
you need to lint your files https://dev.narc.ro/cataclysm/format.html otherwise it can't be merged.

If the chitin arm guards already require joint articulation that is fine then I think what you came up with for leather is a fine solution.

**Comment by bombasticSlacks** [comment]
So this is currently in a spot where I would feel good merging it but because the arm guards are used in crafting tests those are now throwing errors. They basically have an expected amount of time to take and it is take a different amount (since the recipes changed).

**Comment by Balthasar-eu** [comment]
Okay, great.
I changed the ids in the tests now, the crafting times for should still be the same, but the ids changed.

**Comment by MizMahem** [comment]
Maybe a different PR, but I noted a similar issue with leg guards that could maybe take some looking at?

**Comment by Night-Pryanik** [comment]
Could you please resolve conflicts?

**Comment by Balthasar-eu** [comment]
I think there were too many changes in the mean time as that I could easily resolve these conflicts. I also made these changes on a forked master branch, which is not optimal. So I am closing this for now, but might remake it in the future.

**Pull request closed without merging**